concepticondata/conceptlists/Bowern-2021-207a.tsv

```diff
@@ -0,0 +1,208 @@
+Bowern-2021-207a-13	13	ankle	356	body	Basic Vocabulary	803	ANKLE
+Bowern-2021-207a-14	14	armpit	358	body	Basic Vocabulary	1886	ARMPIT
+Bowern-2021-207a-15	15	ash	359	environment	Basic Vocabulary	646	ASH
+Bowern-2021-207a-16	16	ashamed	563	mental	Basic Vocabulary	1894	ASHAMED
+Bowern-2021-207a-17	17	ashes	564	environment	Basic Vocabulary	646	ASH
+Bowern-2021-207a-18	18	at	360	location	Basic Vocabulary	1461	AT
+Bowern-2021-207a-19	19	back	361	body	Basic Vocabulary	1291	BACK
+Bowern-2021-207a-20	20	bad	565	quality	Basic Vocabulary	1292	BAD
+Bowern-2021-207a-21	21	bark	820	environment	Basic Vocabulary	1204	BARK
+Bowern-2021-207a-22	22	be alive	566	body	Basic Vocabulary	1422	BE ALIVE
+Bowern-2021-207a-23	23	belly	363	body	Basic Vocabulary	1251	BELLY
+Bowern-2021-207a-24	24	below	364	location	Basic Vocabulary	1485	BELOW OR UNDER
+Bowern-2021-207a-25	25	big	365	quality	Basic Vocabulary	1202	BIG
+Bowern-2021-207a-26	26	bite	367	body	Basic Vocabulary	1403	BITE
+Bowern-2021-207a-27	27	black	368	colour	Basic Vocabulary	163	BLACK
+Bowern-2021-207a-28	28	blood	369	body	Basic Vocabulary	946	BLOOD
+Bowern-2021-207a-29	29	blow	370	body	Basic Vocabulary	176	BLOW (WITH MOUTH)
+Bowern-2021-207a-30	30	blunt	568	quality	Basic Vocabulary	379	BLUNT
+Bowern-2021-207a-31	31	boil/pimple	372	body	Basic Vocabulary	793	BOIL (OF SKIN)
+Bowern-2021-207a-32	32	bone	373	body	Basic Vocabulary	1394	BONE
+Bowern-2021-207a-33	33	boomerang	569	manufacture	Basic Vocabulary	135	BOOMERANG
+Bowern-2021-207a-34	34	breast	376	body	Basic Vocabulary	1402	BREAST
+Bowern-2021-207a-35	35	breathe	377	body	Basic Vocabulary	1407	BREATHE
+Bowern-2021-207a-36	36	burn	378	environment	Basic Vocabulary	2102	BURN
+Bowern-2021-207a-37	37	chew	379	body	Basic Vocabulary	321	CHEW
+Bowern-2021-207a-38	38	child	380	human	Basic Vocabulary	2099	CHILD
+Bowern-2021-207a-39	39	climb	381	motion	Basic Vocabulary	1102	CLIMB
+Bowern-2021-207a-40	40	cloud	382	environment	Basic Vocabulary	1489	CLOUD
+Bowern-2021-207a-41	41	cold	383	quality	Basic Vocabulary	1287	COLD
+Bowern-2021-207a-42	42	come	384	motion	Basic Vocabulary	1446	COME
+Bowern-2021-207a-43	43	cook	385	impact	Basic Vocabulary	1100	COOK (SOMETHING)
+Bowern-2021-207a-44	44	correct/true	386	quality	Basic Vocabulary	1725	CORRECT (RIGHT)
+Bowern-2021-207a-45	45	count	387	mental	Basic Vocabulary	1420	COUNT
+Bowern-2021-207a-46	46	cry	388	mental	Basic Vocabulary	1839	CRY
+Bowern-2021-207a-47	47	cut	570	impact	Basic Vocabulary	1432	CUT
+Bowern-2021-207a-48	48	cut/hack	389	impact	Basic Vocabulary	155	CHOP
+Bowern-2021-207a-49	49	day	390	time	Basic Vocabulary	1225	DAY (NOT NIGHT)
+Bowern-2021-207a-50	50	die/be dead	391	state	Basic Vocabulary	2101	BE DEAD OR DIE
+Bowern-2021-207a-51	51	dig	392	impact	Basic Vocabulary	1418	DIG
+Bowern-2021-207a-52	52	dingo/wolf	394	fauna	Basic Vocabulary	2461	WOLF-LIKE ANIMAL
+Bowern-2021-207a-53	53	dirty	395	quality	Basic Vocabulary	1230	DIRTY
+Bowern-2021-207a-54	54	dream	397	mental	Basic Vocabulary	1920	DREAM (SOMETHING)
```

Comment:
Because it is not clear whether it is the verb or noun, we could map to DREAMING OR DREAM (as in other Bowern lists).

Reply:
This should be only the verb, based on the Spanish/Portuguese glosses.